paris-tennis.py

```diff
@@ -0,0 +1,178 @@
+
+        # Click Reserver button
+        driver.find_element_by_id('collapse{}{}'.format(name, hour)).find_elements_by_class_name('tennis-court')[
+            0].find_element_by_class_name('button').click()
+        time.sleep(1)
+        # FILL PARTNER INFOS
+        driver.find_element_by_class_name('name').find_element_by_class_name(
+            'form-control').send_keys(PARTNER_NAME)
+        time.sleep(1)
+        driver.find_element_by_class_name('firstname').find_element_by_class_name(
+            'form-control').send_keys(PARTNER_EMAIL)
+        time.sleep(1)
+        driver.find_element_by_id('submitControle').click()
+        time.sleep(1)
+
+        if BotMode is not HOLD_MODE:
+            # PAYMENT AND VALIDATION
+            driver.find_element_by_class_name('price-item').click()
+            time.sleep(1)
+            driver.find_element_by_id('submit').click()
+            logging.info("Booked and payed a reservation at {0}, in {1}, {2} days from today".format(hour, name, day))
+            notification.notify(
+                title='Paris Sport Bot',
+                message="Booked and payed a reservation at {0}, in {1}, {2} days from today".format(hour, name, day),
+                app_icon=None,
+                timeout=10,
+            )
+        else:
+            logging.info(
+                "Held a reservation at {0}, in {1}, {2} days from today - you have 10 minutes to manually pay".format(hour,
+                                                                                                             name,
+                                                                                                             day))
+            notification.notify(
+                title='Paris Sport Bot',
+                message="Held a reservation at {0}, in {1}, {2} days from today - you have 10 minutes to manually pay".format(
+                    hour,
+                    name,
+                    day),
+                app_icon=None,
+                timeout=10,
+            )
+    except Exception as e:
+        logging.error("Failure to book and pay", e)
+
+
+def check_arguments(args):
+    if not (args.Hour and args.Day and args.Name):
+        logging.error("Hour, Day, and Name arguments are mandatory")
+        return False
+
+    if args.Mode:
+        if args.Mode not in BOT_MODES:
+            logging.error("% s is not a valid bot mode" % args.Mode)
+        else:
+            global BotMode
+            BotMode = args.Mode
+
+    if not (int(args.Day) >= 0 and int(args.Day) <= 7):
+        logging.error("% s is not valid day format" % args.Day)
+        return False
+
+    if not (str(args.Hour).endswith("h") and (int(str(args.Hour).split("h")[0]) >= 8
+                                              and int(str(args.Hour).split("h")[0]) <= 20)):
+        logging.error("% s is not valid hour format" % args.Hour)
+        return False
+
+    if not (str(args.Name).replace(" ", "") in PARIS_TENNIS_COURTS):
```

Comment:
probably better to lowercase both input and paris_tennis_courts strings. I'd imagine some users entering the court name all in lowercase.